common/scala/src/main/scala/whisk/common/Logging.scala

```diff
@@ -91,7 +96,12 @@ trait Logging {
  * typically for a TransactionId, the elapsed time in milliseconds and a string containing
  * the given marker token.
  */
-protected case class LogMarker(now: Instant, delta: Long, token: String)
+protected case class LogMarker(token: LogMarkerToken, deltaToStart: Long, deltaForAction: Option[Long] = None) {
```

Comment:
can you update the comment for this class and use javadoc to explain the parameters?
`deltaForAction` should be called something else.